include/usbg/usbg.h

```diff
@@ -220,6 +220,15 @@ typedef enum
 	USBG_FUNCTION_TYPE_MAX,
 } usbg_function_type;
 
+/**
+ * @brief USB OS Descriptor function attributes
+ */
+struct usbg_function_os_desc
+{
+	char compatible_id[USBG_MAX_STR_LENGTH];
```

Comment:
I think we already got rid of constant size strings from our API so I would prefer to not introduce them again;)

Reply:
Ok, I see. I guess we could use dynamic memory allocation for the qw_sign strings too then?

Reply:
Well... as the size of this string is constant (in spec) it's not that important